docs/sources/reference/builder.md

```diff
@@ -204,8 +208,8 @@ to start by **pulling an image** from the [*Public Repositories*](
 multiple images. Simply make a note of the last image ID output by the commit
 before each new `FROM` command.
 
-If no `tag` is given to the `FROM` instruction, `latest` is assumed. If the
-used tag does not exist, an error will be returned.
+If no `tag` or `digest` is given to the `FROM` instruction, the tag `latest` is
```

Comment:
The `tag` or `digest` values are optional. If you omit either of them, the builder assumes a `latest` by default.  The builder returns an error if it cannot match the `tag` value.